File: CLI/GameClasses/LocalizationDb.cs
using System;
using System.Collections.Generic;
using SiiParser;
using SiiParser.Attributes;

namespace SiiParserSCS.GameClasses
{
    [SiiUnit("localization_db")]
    public class LocalizationDb : GameClass
    {
        [SiiAttribute("key[]")]
        public List<Object> Keys { get; set; }
        
        [SiiAttribute("val[]")]
        public List<Object> Values { get; set; }
    }
}

File: src/game-classes/localization-db.ts
import SiiUnit from "../decorators/sii-unit";
import SiiAttribute from "../decorators/sii-attribute";
import GameClass from "./game-class";

@SiiUnit("localization_db")
class LocalizationDb extends GameClass {
    namespace: string;

    @SiiAttribute("val[]")
    public values: string[];

    @SiiAttribute("key[]")
    public keys: string[];
}

export default LocalizationDb;